sdk/hdinsight/Azure.ResourceManager.HDInsight/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0 (2023-06-20)
```

Comment:
Please provide a new date

Reply:
Update. Please help release this[ PR](https://github.com/Azure/azure-sdk-for-net/pull/37199) to release version 1.1.0, and then release this version 1.1.1.